src/main/java/de/tum/in/www1/artemis/config/migration/entries/MigrationEntry20230808_203400.java

```diff
@@ -68,7 +68,7 @@ public class MigrationEntry20230808_203400 extends MigrationEntry {
 
     private final CopyOnWriteArrayList<ProgrammingExerciseParticipation> errorList = new CopyOnWriteArrayList<>();
 
-    private static final List<String> MIGRATABLE_PROFILES = List.of("bamboo", "gitlab", "jenkins");
+    private static final List<String> MIGRATABLE_PROFILES = List.of("bamboo", "jenkins");
```

Comment:
Are you sure that you can simply remove this here?

Reply:
As far as I have seen, initially all programming exercise profiles were added to the list. The migration only is compatible with Bitbucket/Bamboo and GitLab/Jenkins. All other profiles were removed in https://github.com/ls1intum/Artemis/pull/7223. As the Bamboo profile is sufficient for the Bitbucket/Bamboo setup, I assume that Jenkins is sufficient for the GitLab/Jenkins setup - especially as it results in problem with GitLab CI.